composeApp/src/commonMain/kotlin/band/effective/office/elevator/ui/employee/allEmployee/EmployeeScreen.kt

```diff
@@ -232,19 +219,33 @@ fun EveryEmployeeCard(emp: EmployeeCard, onCardClick: () -> Unit) {
         shape = RoundedCornerShape(12.dp),
         modifier = Modifier
             .fillMaxSize()
-            .padding(5.dp)
+            .padding(0.dp,0.dp,0.dp,15.dp)
```

Comment:
use named parameters, otherwise it is not clear what to relate to what.
Example:
`Modifier.padding(horizontal = 16.dp)`